Bottom: 84de4d63253c2fd5283423ac0afbb605c3a3f53d
Top:    e67e297f4f5bc6a18100667adaaa1efaea834976
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 23:18:41 +0000

Refresh of extend-board-export

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 2809a69..f219c6b 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -255,6 +255,9 @@ object3d_from_contours (const POLYAREA *contours,
   int offset_in_ct;
   int ct_npoints;
 
+  if (contours == NULL)
+    return NULL;
+
   /* Loop over all board outline pieces */
   pa = contours;
   do {
